### DISCUSSION TITLE
Hide User Email?

### DISCUSSION THREAD

**Discussion by utkonos** [Vector35/binaryninja-api#5005] [Ideas] [answered]
In the About Binary Ninja window, the serial number is obscured. Could the user's email address also be obscured as well? If so, it may be useful to obscure it using a different UI behavior: reveal on click rather than copy on click.

**Comment by fuzyll** [comment, marked as answer]
Yeah, we can do that. Good suggestion. I turned this into an issue (#5007) so we can track it.
